pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1669,6 +1669,18 @@ in
 
   nvim-treesitter-parsers = lib.recurseIntoAttrs self.nvim-treesitter.grammarPlugins;
 
+  nvim-treesitter-sexp = super.nvim-treesitter-sexp.overrideAttrs {
+    dependencies = with self; [
+      (nvim-treesitter.withPlugins (p: [
```

Comment:
@GaetanLepage This actually got me thinking about our discussion in matrix.. Will we run into issues with plugins that declare their own treesitter that might conflict with other treesitters in rtp?

Reply:
Isn't it better to allow the user to install the grammar instead of declaring them here?

Reply:
I guess I'd be curious if the nvimRequireCheck passes without declaring the treesitter dependency.

Reply:
@khaneliman it works a-okay. @abhisheksingh0x558 I'll get rid of the nvim-treesitter dep